core/controlplane/config/templates/cluster.yaml

```diff
@@ -468,6 +468,12 @@ worker:
 #      kube2IamSupport:
 #        enabled: true
 #
+#      # Propagate custom CLI options to kubelet
+#      # Provided example enables docker image garbage collection as in https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/
```

Comment:
ok, then they are too high by default ,right. Want me to change formulation in the comment ?